<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgTextureLoadOptions</class>
 <widget class="QDialog" name="CQDlgTextureLoadOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texture Load Options</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="qqOk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="qqScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Scale textures to</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="qq256">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>max. 256 x 256</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qq512">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>max. 512 x 512</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qq1024">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>85</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>max. 1024 x 1024</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qq2048">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>max. 2048 x 2048</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qq128">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>max. 128 x 128</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqScale</tabstop>
  <tabstop>qq128</tabstop>
  <tabstop>qq256</tabstop>
  <tabstop>qq512</tabstop>
  <tabstop>qq1024</tabstop>
  <tabstop>qq2048</tabstop>
  <tabstop>qqOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
